dashboard/app/models/levels/javalab.rb

```diff
@@ -71,6 +71,9 @@ def non_blockly_puzzle_level_options
       # set it to nil here and let other code put it in app_options
       level_prop['teacherMarkdown'] = nil
 
+      # Set the javabuilder url
+      level_prop['javabuilderUrl'] = CDO.javabuilder_url
```

Comment:
In the future, we might want to combine this with a DCDO key in case we want to change at runtime which release/deployment of the Javabuilder service we want a Code Studio environment to use without waiting for a build.